CHANGELOG.md

```
@@ -19,6 +19,229 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Added `getPositionExpandRentExemption` function. It return minimum balance required to pay for a position account expansion for rent exemption.
+- Added `quoteExtendPosition` function. It return minimum balance required to pay for rent exemption for the given position expansion and bin array accounts to cover the expanded position range.
+- Added `decreasePositionLength` function. It return transaction to shrink an expanded position. The rent is not returned upon decrement, but returned when the position is closed.
+- Added `increasePositionLength` function. It return transaction to expand a position up to maximum 1400 bins.
+- Added `simulateRebalancePosition` function. It return a locally simulated rebalanced position, and required parameters for on chain rebalancing.
+- Added `createExtendedEmptyPosition` function. It return transaction to create an empty position with maximum 1400 bins.
+- Added `simulateRebalancePositionWithStrategy` function. It return a locally simulated rebalanced position with current price rebalanced to the center of the position.
+- Added `rebalancePosition` function. It return instructions to rebalance a position.
+- Added `initializeMultiplePositionAndAddLiquidityByStrategy` function. It allow user to initialize multiple extendable positions that cover wide bin range without chainsaw issue.
+- Added `addLiquidityByStrategyChunkable` function. It allow user to add liquidity to single extendable positions that cover wide bin range without chainsaw issue.
+
+### Changed
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)